Added Sampling-Randomization and HFCs Checklists (#45)
worldbank/dime-standards: dime-coding-standards/checklists/tex/hfcs_checklist.tex

\newcommand{\dimeTextField}[3]{\TextField[name=#1, height=0.3cm, width=#2, bordercolor=gray]{#3}}

\newcommand{\titleBox}[1]{
	\begin{tcolorbox}
		[colframe = fontcolor,
		colback = fontcolor,
		sharp corners,
		halign = flush center,
		valign = center,
		height = 0.3\textwidth,
		after skip = 1cm]
		#1
	\end{tcolorbox}
}

\begin{document}
	\begin{fullwidth}
		
		\titleBox{
			\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Peer Code Review - High-Frequency Checks Checklist}} \\
				\Large\textbf{{v1.0}}}
		}

  \dimeTextField{time}{2.5 cm}{Reviewer:} 

        \dimeTextField{time}{2.5 cm} {Coder:}
		
		\textbf{NOTE:} Make sure that fill out this checklist \textbf{ONLY IF} your partner's submission includes request to review \textbf{high-frequency checks}. 
		
		\section*{High-frequency checks}
  This checklist lists important factors to consider while reviewing your code review partner's \textbf{high-frequency checks}. Please fill this checklist, and submit it as an attachment when you submit \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed form.}

  \hfill \break 

>   \textbf{Note:} We are piloting this checklist as part of improving the code review process. The questions in this list are therefore \textbf{not required}, and we welcome your feedback on the points listed below.

Reviewer: > The questions in this list are therefore
What do you refer to as "this list" here? I think it is ambiguous if you refer to the two questions in the "General" subsection or if you mean all questions in this document.

  \subsection*{General}
		
		\begin{itemize}
			\setlength\itemsep{-0.1em}
			\item[] \dimeCheckBox{The data shared does not contain any personally-identifying information (PII).}

                \item[] \dimeCheckBox{The code creates a tracking sheet (or dashboard) to track progress of the survey.}
        \end{itemize}

 \hfill \break 

> The following sections cover various aspects of the \textbf{contents of the high-frequency checks} included in the package, but not all may apply to the high-frequency checks you are reviewing.

Reviewer: I find it a bit tricky to follow what description goes to what section.

Consider making `\subsection*{General}` a section instead of a subsection. Call it `\section*{General Questions}` , then before this row have a new section `\section*{HFC Questions}`.

> The following sections cover various aspects of the \textbf{contents of the high-frequency checks} included in the package, but not all may apply to the high-frequency checks you are reviewing.

Reviewer 2: what is meant by 'included in the package'?

\subsection*{Survey progress}
Tracking \textbf{survey progress} involves checks for the following:
            \begin{itemize}

            \item[]\dimeCheckBox{\textbf{Response rates -}} 
            The high-frequency checks contain/export:
            
            \begin{itemize}
            

>                 \item[] \dimeCheckBox{Table on number/share of surveys attempted, completed, and declined.}

Reviewer 2: i don't think we need to specify format (table, bar graph, etc). So here the important poitn is that they are tracking number/share of surveys attempted/completed/declined but it doesn't have to be in a table. same for all items.

                
                \item[] \dimeCheckBox{Bar graphs for surveys attempted, completed, and declined - for each day}
                
                \item[] \dimeCheckBox{Graph for total surveys attempted, completed, and declined - on aggregate, and/or over last \textbf{X} days}
                
                \item[] \dimeCheckBox{Bar graph for reasons for declining (categorical)}
                
            \end{itemize}

            \item[]\dimeCheckBox{\textbf{Duration -}} 

>             The high-frequency checks contain/export:

Reviewer 2: i'm not clear on the difference between this group and the previous group

Author: Updated language to clarify the difference - the previous group refers to duration of each interview in general, while this group deals with interview durations further split up by enumerator

            
            \begin{itemize}
            
                \item[] \dimeCheckBox{Bar graph of average duration for each day}
                

>                 \item[] \dimeCheckBox{Distribution of duration (either as k-desnity curves, or as box-plots by modules/sections}

Reviewer: ```suggestion
                \item[] \dimeCheckBox{Distribution of duration (either as k-desnity curves, or as box-plots by modules/sections)}
```

                
            \end{itemize}

            \end{itemize}
            

\subsection*{Survey quality}
Tracking \textbf{survey quality} involves checks for the following:
            \begin{itemize}
            
            \item[]\dimeCheckBox{\textbf{Duplicates -}} 
            The high-frequency checks contain/export:
            
            \begin{itemize}
            

>                 \item[] \dimeCheckBox{Spreadsheet listing duplicate entries in ID and non-ID variables to share with the field team for corrections}

Reviewer: Does "non-ID variables to" refer to the list of non-ID variables for which two duplicates have different values? The thing `iecompdup` and `ieduplicates` do?

If it is, consider making this two distinct points. As listing duplicates in ID variables lists duplicates, and the other provide info on how to resolve them. They are very much related but you the first one can be done without the second, even though we recommend doing both. 
```suggestion
                \item[] \dimeCheckBox{Spreadsheet listing duplicate entries in ID}
                \item[] \dimeCheckBox{Spreadsheet listing some information to be used to resolve duplicates, such as listing non-ID variables for which duplicates has different values}
```

Reviewer 2: agree. one point is whether they are checking for duplicates, the other is whether they are exporting duplicates so that they can be reviewed in the field.

                
                \item[]\dimeCheckBox{Spreadsheet listing the errors in the data that need to be shared with the field team for corrections}
                
                \item[]\dimeCheckBox{Bar graphs/k-density curves for \textbf{continuous variables}}
                
                \item[]\dimeCheckBox{Graph for number of missing and extended missing responses (like \texttt{Do not know}, \texttt{Refuse to answer}) by question type }
                
                \item[]\dimeCheckBox{Graph for number of outliers by question type}
                
            \end{itemize}

            \dimeCheckBox{\textbf{Inconsistency in responses -}} 
            The high-frequency checks contain/export:
            \begin{itemize}
            
                \item[]\dimeCheckBox{Spreadsheets listing inconsistent responses with questions and their values that needs to be shared with the field team for corrections. For example, inconsistencies across variables for the same respondent, or for the same variables across different time periods.}
                
            \end{itemize}

>             \dimeCheckBox{\textbf{Form version used -}} 
>             The high-frequency checks contain/export:
>             \begin{itemize}
>             
>                 \item[]\dimeCheckBox{Table listing number of responses by form-version and date}
>                 
>             \end{itemize}

Reviewer: ```suggestion
            \textbf{Form version used -}
            The high-frequency checks contain/export:
            \begin{itemize}
            
                \item[]\dimeCheckBox{Table listing number of responses by form-version and date}
                
            \end{itemize}
```
I am not sure when and how the check box in the upper level will be checked when the item in the list wont or vice versa. This could lead to confusion. I suggest removing the check boxes for the title item have check boxes only for `\items[]`.

            \end{itemize}

\subsection*{Enumerator productivity}
Tracking \textbf{enumerator productivity} involves checks for the following:

            \begin{itemize}

            \item[]\dimeCheckBox{\textbf{Response rates -}} 
            The high-frequency checks contain/export:
            
            \begin{itemize}
            
                \item[] \dimeCheckBox{Table on number/share of surveys attempted, completed, and declined - for each enumerator}
                
                \item[] \dimeCheckBox{Table on number of attempts made to contact each interviewee - for each enumerator}
                
                \item[] \dimeCheckBox{Graph for total surveys attempted, completed, and declined - for each enumerator}
                
            \end{itemize}

            \item[]\dimeCheckBox{\textbf{Duration -}} 
            The high-frequency checks contain/export:
            
            \begin{itemize}
            
                \item[] \dimeCheckBox{Bar graph of average duration over each enumerator}
                

>                 \item[] \dimeCheckBox{Box plots of duration for each module over each enumerator}

Reviewer 2: this is too detailed. just need to review whether they are checking duration by enumerator but i don't think we need to mandate how they are doing that.
